python/lsst/cp/pipe/defects.py

```diff
@@ -716,3 +858,70 @@ def run(self, inputDefects, camera):
         return pipeBase.Struct(
             mergedDefects=merged,
         )
+
+# Subclass the MergeDefects task to reduce the input dimensions
+# from ("instrument", "detector", "exposure") to
+# ("instrument", "detector").
+
+
+class MergeDefectsCombinedConnections(MergeDefectsConnections,
+                                      dimensions=("instrument", "detector")):
+    inputDefects = cT.Input(
+        name="cpPartialDefectsFromDarkCombined",
+        doc="Measured defect lists.",
+        storageClass="Defects",
+        dimensions=("instrument", "detector",),
+        multiple=False,
+    )
+    secondaryInputDefects = cT.Input(
+        name="cpPartialDefectsFromFlatCombinedWithFilter",
+        doc="Additional measured defect lists.",
+        storageClass="Defects",
+        dimensions=("instrument", "detector", "physical_filter"),
+        multiple=False,
```

Comment:
Right now we only send one flat in, so this `multiple=False` is okay.  I think this is okay, but I would like to know from @jchiang87 or @czwa if there is any possibility that we would be sending multiple combined flats from different filters through this defect pipeline.

Reply:
I can't think of any reason for sending multiple combined flats through this pipeline.